### PR TITLE
Booking details

### PR DESCRIPTION
**Pull request by redvelvet511**
Added validations, and link to Superpower ID that they can click to view details

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -11,8 +12,16 @@ def create
...
     end
   end
-  
+
   def index
     @bookings = Booking.all
   end
+
+  def my_bookings
+    @bookings = Booking.all
```
Comment (Komezu) [inline app/controllers/bookings_controller.rb]: should only return bookings of current user

Comment (Komezu) [reply app/controllers/bookings_controller.rb]: could have @bookings_as_rentee and @bookings_as_rentor (two instance variables), that we'll use in two different sections in the view

File: app/views/bookings/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<h1>Booking Details</h1>
+
+<div class="cards">
+  <p><%= "Superpower ID: "%><%= link_to "#{@booking.superpower_id}", "/superpower_path" %></p>
```
Comment (Komezu) [inline app/views/bookings/show.html.erb]: you don't need a <%= %> tag for the superpower ID part, as it's not ruby code

**Review by Komezu — CHANGES_REQUESTED** [review]
Good job :)

File: app/views/bookings/show.html.erb (hunk 2 above)
Comment (redvelvet511) [reply app/views/bookings/show.html.erb]: done :)

**Review by Komezu — APPROVED** [review]

**Pull request merged**